### DISCUSSION TITLE
sidebar and urlbar:  size (icons and width) customization's?

### DISCUSSION THREAD

**Discussion by dashingdon** [zen-browser/desktop#1163] [Ideas]
Is there a way to reduce the size of the icons and the width of the urlbar / sidebar?  I would prefer a smaller icon size and width to optimize space. 

PS: I have attempted the CSS mod approach, but it was never successful for me. Maybe someone can guide me in the right direction?

For comparison, below are the screenshots from the current configuration between Zen and Vivaldi.

thank you.

- `zen browser` 

![image](https://github.com/user-attachments/assets/e242aa8d-9227-40d3-9cc6-8176fb067c9b)

- `vivaldi` 

![image](https://github.com/user-attachments/assets/e00dbb45-1d0a-44ee-828d-e63f23a6dce1)

**Comment by Seigiard** [comment]
You can use custom CSS file and edit firefox/zen UI using css

https://github.com/zen-browser/desktop/discussions/1923
https://www.reddit.com/r/FirefoxCSS/comments/vpg9k9/how_do_i_edit_firefoxs_ui/

**Comment by CrackRabb1t** [comment]
The Zen team is moving towards an apple touch screen model.  Older version are good  before Jan 2025 after not so much. There is a new browser called Floorp that matches what you like with high customization and privacy base on Firefox.
